src/IROperator.cpp

```diff
@@ -92,6 +92,12 @@ const int64_t *as_const_int(Expr e) {
         return NULL;
     } else if (const Broadcast *b = e.as<Broadcast>()) {
         return as_const_int(b->value);
+    } else if (const Cast *c = e.as<Cast>()) {
```

Comment:
Also they're not correct. as_const_uint(Cast(UInt(8), 3456), ptr) will return the wrong thing.